appveyor.yml

```diff
@@ -19,14 +19,17 @@ install:
   - SET PATH=C:\Ruby%ruby_version%\bin;%PATH%
   - echo %PATH%
   - ruby --version
-  - gem update --system
+  - gem update --system || gem update --system || gem update --system
+  - gem install bundler --quiet --no-ri --no-rdoc || gem install bundler --quiet --no-ri --no-rdoc || gem install bundler --quiet --no-ri --no-rdoc
+  - update_rubygems
   - gem --version
-  - gem install bundler --no-ri --no-rdoc
-  - SET BUNDLE_GEMFILE=Gemfile.windows
   - bundler --version
+  - SET BUNDLE_IGNORE_CONFIG=true
+  - SET BUNDLE_FROZEN=1
+  - SET BUNDLE_WITHOUT=development:guard:maintenance:tools:integration:changelog:docgen:travis:style:omnibus_package:aix:bsd:linux:mac_os_x:solaris
 
 build_script:
-  - bundle install --without no_windows omnibus_package
+  - bundle install || bundle install || bundle install
```

Comment:
Yeah, this is copied straight from the chef appveyor.yml (trying to keep them the same where possible).